samza-core/src/main/java/org/apache/samza/config/JobCoordinatorConfig.java

```diff
@@ -23,7 +23,6 @@
 
 public class JobCoordinatorConfig extends MapConfig {
   public static final String JOB_COORDINATOR_FACTORY = "job-coordinator.factory";
```

Comment:
I created a separate JIRA for it - https://issues.apache.org/jira/browse/SAMZA-1224 This is a good reminder :)
I will put a patch for that config today. let's not address it in this PR.